mappings/net/minecraft/block/enums/Instrument.mapping

```diff
@@ -1,7 +1,15 @@
 CLASS net/minecraft/unmapped/C_onjtjtsi net/minecraft/block/enums/Instrument
 	FIELD f_jpklnqjf sound Lnet/minecraft/unmapped/C_avavozay;
+	FIELD f_sjascaer isMobHead Z
```

Comment:
```suggestion
	FIELD f_sjascaer mobHead Z
```

Reply:
looks like I need to reread the conventions